app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```

Comment:
I don't mind this for now but you might want to point your `root_path` to another controller (that deals with restaurants) later.